core/assets/bundles/bundle_fr.properties

```
@@ -174,34 +176,34 @@ data.exported = Données Exportées.
-quit.confirm = Êtes-vous sûr de quitter?
-quit.confirm.tutorial = Êtes-vous sur de ce que vous faites ?\n Le tutoriel peut être repris dans [accent]Paramètres->Jeu->Reprendre le turoriel.[]
+classic.export.text = [accent]Mindustry[] vient d'avoir une mise à jour majeure.\nDes sauvegardes et/ou des cartes de la version Classic (v3.5 build 40) ont été détectées. Souhaitez-vous exporter ces sauvegardes dans le dossier accueil de votre télephone, pour les utiliser dans Mindustry Classic ?
+quit.confirm = Êtes-vous sûr de vouloir quitter?
+quit.confirm.tutorial = Êtes-vous sur de ce que vous faites?\nLe tutoriel peut être repris dans [accent]Paramètres->Jeu->Reprendre le tutoriel.[]
 loading = [accent]Chargement...
 saving = [accent]Sauvegarde...
 wave = [accent]Vague {0}
-wave.waiting = [LIGHT_GRAY]Vague dans {0}
-wave.waveInProgress = [LIGHT_GRAY]Wave en cours
-waiting = [LIGHT_GRAY]En attente...
+wave.waiting = [lightgray]Vague dans {0}
+wave.waveInProgress = [lightgray]Wave en cours
+waiting = [lightgray]En attente...
 waiting.players = En attente de joueurs...
-wave.enemies = [LIGHT_GRAY]{0} Ennemis restants
-wave.enemy = [LIGHT_GRAY]{0} Ennemi restant
+wave.enemies = [lightgray]{0} Ennemis restants
+wave.enemy = [lightgray]{0} Ennemi restant
 loadimage = Charger l'image
 saveimage = Sauvegarder l'image
 unknown = Inconnu
 custom = Personnalisé
 builtin = Intégré
-map.delete.confirm = Êtes-vous sûr de supprimer cette carte? Cette action ne peut pas être défaite!
+map.delete.confirm = Voulez vous vraiment supprimer cette carte? Cette action ne peut pas être annulée!
 map.random = [accent]Carte aléatoire
-map.nospawn = Cette carte n'a pas de base pour qu'un joueur puisse y apparaisse! Veiller ajouter une [ROYAL]base bleue[] sur cette carte dans l'éditeur.
-map.nospawn.pvp = Cette carte n'a pas de base ennemies pour qu'un joueur ennemi y apparaisse! Ajouter au moins une [SCARLET]base rouge[] sur cette carte dans l'éditeur.
-map.nospawn.attack = Cette carte n'a aucune base ennemie à attaquer! Veuillez ajouter une [SCARLET]base rouge[] sur cette carte dans l'éditeur.
+map.nospawn = Cette carte n'a pas de base pour qu'un joueur puisse y apparaisse! Ajoutez une base[accent] orange[] sur cette carte dans l'éditeur.
+map.nospawn.pvp = Cette carte n'a pas de base ennemies pour qu'un joueur ennemi y apparaisse! Ajouter au moins une base [SCARLET] non-orange[] dans l'éditeur.
+map.nospawn.attack = Cette carte n'a aucune base ennemie à attaquer! Veuillez ajouter une base[SCARLET] rouge[] sur cette carte dans l'éditeur.
 map.invalid = Erreur lors du chargement de la carte: carte corrompue ou invalide.
 editor.brush = Pinceau
 editor.openin = Ouvrir dans l'éditeur
 editor.oregen = Génération de minerais
 editor.oregen.info = Génération de minerais:
-editor.mapinfo = Infos sur la carte
+editor.mapinfo = Infos carte
```

[Pour plus de consistance avec le fichier original](https://github.com/Anuken/Mindustry/blob/master/core/assets/bundles/bundle.properties#L205)
```suggestion
editor.mapinfo = Infos Carte
```